container/ubuntu16_04/bazel/BUILD

```diff
@@ -0,0 +1,65 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
call this  
container/ubuntu16_04/builders/BUILD

Reply:
Move this file to container/ubuntu16_04/builders/bazel/BUILD